build/cmake/lib/webview/CMakeLists.txt

```diff
@@ -65,23 +67,28 @@ elseif(WXMSW)
                 WORKING_DIRECTORY ${WEBVIEW2_PACKAGE_DIR}
             )
         endif()
+        set(WEBVIEW2_PACKAGE_DIR ${WEBVIEW2_PACKAGE_DIR} CACHE INTERNAL "" FORCE)
 
         wx_lib_include_directories(wxwebview PRIVATE "${WEBVIEW2_PACKAGE_DIR}/build/native/include")
         if(NOT MSVC)
             wx_lib_include_directories(wxwebview PRIVATE "${wxSOURCE_DIR}/include/wx/msw/wrl")
-            target_compile_options(wxwebview PRIVATE -Wno-unknown-pragmas)
+            if (NOT wxBUILD_MONOLITHIC)
+                target_compile_options(wxwebview PRIVATE -Wno-unknown-pragmas)
+            endif()
         endif()
 
-        if (CMAKE_SIZEOF_VOID_P EQUAL 4)
-            set(WEBVIEW2_ARCH x86)
-        else()
-            set(WEBVIEW2_ARCH x64)
+        if (NOT wxBUILD_MONOLITHIC)
+            if (CMAKE_SIZEOF_VOID_P EQUAL 4)
+                set(WEBVIEW2_ARCH x86)
+            else()
+                set(WEBVIEW2_ARCH x64)
+            endif()
+
+            add_custom_command(TARGET wxwebview POST_BUILD
+                COMMAND ${CMAKE_COMMAND} -E copy
+                "${WEBVIEW2_PACKAGE_DIR}/build/native/${WEBVIEW2_ARCH}/WebView2Loader.dll"
+                "$<TARGET_FILE_DIR:wxwebview>/WebView2Loader.dll")
```

Comment:
Sorry if I'm missing something obvious to CMake-literal people, but it seems to me like we're not copying the DLL any longer when using monolithic wx build. If this is indeed the case, why don't we do it? This loader DLL should be still needed, even when there is no separate wxwebview library, shouldn't it?

Reply:
Correct, now it is not copied for the monolithic build.
The `wxmono` target does not exist when adding the custom command to the target. And when the `wxmono` target is created earlier, it can not be used because it is in a different directory.

But I found a possible solution by moving `add_custom_command()` to a separate function that is called after creating the `wxmono` library. I still need to test if it actually copies the library correctly.